teps/0092-scheduling-timeout.md

```diff
@@ -211,17 +213,17 @@ field would be deprecated once this feature is beta.
 ```yaml
 spec:
   timeouts:
-    scheduling: "5m" # this is the new option
-    execution: "10m" # this option would have the same functionality as the existing `timeout` field
+    scheduling: "5m" # optional, if specified, from startTime to scheduledTime
+    execution: "5m" # optional, if specified, from scheduledTime to completionTime
+    total: "10m" # this option would have the same functionality as the existing `timeout` field
```

Comment:
It would be helpful to give a few examples of how these fields will be validated/calculated/defaulted, based on what the user sets. Maybe in a new "examples" section?